package.json

```diff
@@ -4,16 +4,38 @@
   "license": "MIT",
   "author": "The Ilios Team (info@iliosproject.org)",
   "scripts": {
-    "lint": "pnpm --filter '*' lint",
-    "lint:fix": "pnpm --filter '*' lint:fix",
-    "test": "pnpm --filter '*' --workspace-concurrency=1 test",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:css": "stylelint \"**/*.scss\"",
+    "lint:css:fix": "stylelint \"**/*.scss\" --fix",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "start:docs": "pnpm run --filter docs start",
     "start:test-app": "pnpm run --filter test-app start",
+    "test:docs": "pnpm run --filter docs test",
     "test:test-app": "pnpm run --filter test-app test",
-    "start:docs": "pnpm run --filter docs start",
     "preinstall": "npx only-allow pnpm"
   },
   "devDependencies": {
-    "pre-commit": "^1.2.2"
+    "@babel/eslint-parser": "7.23.10",
+    "@babel/plugin-proposal-decorators": "^7.23.9",
+    "concurrently": "^8.2.2",
+    "ember-template-lint": "^5.13.0",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-ember": "^12.0.2",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-qunit": "^8.1.1",
+    "eslint-plugin-yml": "^1.12.2",
+    "pre-commit": "^1.2.2",
```

Comment:
Pre-commit doesn't actually work with `pnpm` this can go along with the `pre-commit` lint specifier.

Reply:
removed.

:white_check_mark: